Formula/micronaut@2.rb

```diff
@@ -0,0 +1,24 @@
+class MicronautAT2 < Formula
+  desc "Modern JVM-based framework for building modular microservices"
+  homepage "https://micronaut.io/"
+  url "https://github.com/micronaut-projects/micronaut-core/releases/download/v2.0.0.M1/micronaut-2.0.0.M1.zip"
+  version "2.0.0.M1"
```

Comment:
we dont ship with milestone releases, can we wait until it becomes more official?

Reply:
👌